src/canvas.tsx

```diff
@@ -584,18 +584,18 @@ export const useCanvas = (props: UseCanvasProps): DomEventHandlers => {
       ...props
     } = state.current
     sharedState.current = props
-  }, [size, defaultCam])
+  }, [size, defaultCam]) // is defaultCam intentional?
```

Comment:
here is possible missing hook dependency. please clarify intention

Reply:
updates the shared state model when the camera changes, which triggers all consumers. it's correct